scripts/sct_maths.py

```diff
@@ -114,16 +114,18 @@ def get_parser():
     thresholding.add_argument(
         "-adap",
         metavar=Metavar.list,
+        type=list_type(',', int),
         help="R|Threshold image using Adaptive algorithm (from skimage). Separate following arguments with ',':"
              "\n Block size: Odd size of pixel neighborhood which is used to calculate the threshold value (e.g. 3, 7, 21, ...)"
```

Comment:
just curious: is that OK to have a space after the comma? if not, we might want to update the examples

Reply:
In this specific case, it's okay, as `int(" 7")` for example will evaluate to `7`. See below for a quick test I did:

```
>>> from spinalcordtoolbox.utils import list_type
>>> str_with_spaces = "3, 7, 21"
>>> type_func = list_type(',', int)
>>> type_func(str_with_spaces)
[3, 7, 21]
```

In other cases (e.g. a list of strings) it could be an issue, though. So, we might want to make the change anyway, just to encourage a consistent input format.

Reply:
@jcohenadad I've just realized that we both misintepreted! The text `(e.g. 3, 7, 21, ...)` above isn't a list, but actually 3 individual examples of Block size, which only needs a single value. 

I have rewritten the usage to be clearer in d443363b1028fe5fc653abd8c3b06b8af9913d9d

Reply:
oh geeze! you're right.  Wow, that was really not a good usage example! (i'm authorized to say that because i was the one writing the usage at the first place 😬 )